karekkhaleb/Questioner
# #162747441 differentiate admin and normal users

#### What does this PR do?
it defines specific actions that belong and the ones for a normal user.
#### Description of Task to be completed?
* give the admin the ability to create a tag and also add tags to a meetup
* give the admin the ability to see all the questions asked for a given meetup and their comments
* let the admin see all the people who are attending a particular meetup
#### How should this be manually tested?
> To log in as the admin use email `admin@mysite.com` and password: `admin`
```
* on the admin's dashboard, click where you see the number of users to see them
* click the number of questions to see them
```
> To log in as a regular user, use email: `user@mysite.com` and password: `user`
#### What are the relevant pivotal tracker stories?
#162747441
